<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="Compare">
     <attribute name="title">
      <string>Compare</string>
     </attribute>
     <widget class="QRadioButton" name="srtRadioButton">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>140</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>srt</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="txtRadioButton">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>140</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>120</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>120</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>120</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>txt</string>
      </property>
     </widget>
     <widget class="QPushButton" name="comparePushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>550</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    color: #333;
    border: 1px solid #555;
    border-style: outset;
    background:white;
    padding: 5px;
    }

QPushButton:pressed {
    background: #0078d7
    }</string>
      </property>
      <property name="text">
       <string>Compare</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>79</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>2nd file:</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="firstFile">
      <property name="geometry">
       <rect>
        <x>87</x>
        <y>43</y>
        <width>455</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit{
	border:1px solid #555
}</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="secondFile">
      <property name="geometry">
       <rect>
        <x>87</x>
        <y>79</y>
        <width>455</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit{
	border:1px solid #555
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>43</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1st file:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fileDialog1">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>53</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    color: #333;
    border: 1px solid #555;
    border-style: outset;
    background:white;
    padding: 5px;
    }

QPushButton:pressed {
    background: #0078d7
    }</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="fileDialog2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>89</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    color: #333;
    border: 1px solid #555;
    border-style: outset;
    background:white;
    padding: 5px;
    }

QPushButton:pressed {
    background: #0078d7
    }</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="csvRadioButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>140</y>
        <width>50</width>
        <height>30</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>120</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>120</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Highlight">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>120</green>
            <blue>215</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>csv</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Sort">
     <attribute name="title">
      <string>Sort</string>
     </attribute>
     <widget class="QPushButton" name="sortPushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>550</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    color: #333;
    border: 1px solid #555;
    border-style: outset;
    background:white;
    padding: 5px;
    }

QPushButton:pressed {
    background: #0078d7
    }</string>
      </property>
      <property name="text">
       <string>Sort</string>
      </property>
     </widget>
     <widget class="QLabel" name="convertInfoLabel_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>550</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Put all .srt files in the same folder with this program and press the Sort Button.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Convert">
     <attribute name="title">
      <string>Convert</string>
     </attribute>
     <widget class="QPushButton" name="convertPushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>550</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    color: #333;
    border: 1px solid #555;
    border-style: outset;
    background:white;
    padding: 5px;
    }

QPushButton:pressed {
    background: #0078d7
    }</string>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QLabel" name="convertInfoLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>550</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Put all .srt and .vtt files in the same folder with this program and press the Convert Button.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Clean">
     <attribute name="title">
      <string>Clean</string>
     </attribute>
     <widget class="QPushButton" name="cleanPushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>550</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    color: #333;
    border: 1px solid #555;
    border-style: outset;
    background:white;
    padding: 5px;
    }

QPushButton:pressed {
    background: #0078d7
    }</string>
      </property>
      <property name="text">
       <string>Clean Srt and Generate Txt</string>
      </property>
     </widget>
     <widget class="QLabel" name="convertInfoLabel_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>550</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Put all .srt files in the same folder with this program and press the Clean Srt and Generate Txt Button.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Validator">
     <attribute name="title">
      <string>Validator</string>
     </attribute>
     <widget class="QPushButton" name="validatePushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>550</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    color: #333;
    border: 1px solid #555;
    border-style: outset;
    background:white;
    padding: 5px;
    }

QPushButton:pressed {
    background: #0078d7
    }</string>
      </property>
      <property name="text">
       <string>Validate</string>
      </property>
     </widget>
     <widget class="QLabel" name="convertInfoLabel_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>550</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Put all .srt files in the same folder with this program and press the Validate button.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTextEdit" name="validateFeedbackTextEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>550</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SequenceCheck">
     <attribute name="title">
      <string>Sequence Check</string>
     </attribute>
     <widget class="QTextEdit" name="checkSequenceFeedbackTextEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>550</width>
        <height>200</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="checkSequencePushButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>550</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    color: #333;
    border: 1px solid #555;
    border-style: outset;
    background:white;
    padding: 5px;
    }

QPushButton:pressed {
    background: #0078d7
    }</string>
      </property>
      <property name="text">
       <string>Check Sequence</string>
      </property>
     </widget>
     <widget class="QLabel" name="convertInfoLabel_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>550</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Put all .srt files in the same folder with this program and press the Check Sequence button.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="srtPrep">
     <attribute name="title">
      <string>STR Prep</string>
     </attribute>
     <widget class="QPushButton" name="checkSequencePushButton_2">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>190</y>
        <width>275</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    color: #333;
    border: 1px solid #555;
    border-style: outset;
    background:white;
    padding: 5px;
    }

QPushButton:pressed {
    background: #0078d7
    }</string>
      </property>
      <property name="text">
       <string>Check Sequence</string>
      </property>
     </widget>
     <widget class="QPushButton" name="checkSequencePushButton_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>275</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Light</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    color: #333;
    border: 1px solid #555;
    border-style: outset;
    background:white;
    padding: 5px;
    }

QPushButton:pressed {
    background: #0078d7
    }</string>
      </property>
      <property name="text">
       <string>Prep STR</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
